index.html

```diff
@@ -28,61 +28,6 @@ <h1 class="main-title">Light Your Heart on Fire</h1>
 
   </section>
 
-
-  <!-- Begin Mailchimp Signup Form -->
-
-  <section class="hero">
-    <div class="newsletter-signup">
-
-      <div id="mc_embed_signup">
-      <form action="https://gmail.us5.list-manage.com/subscribe/post?u=5b441666cd710919cbcaa86e8&amp;id=d37fc91c90" method="post" id="mc-embedded-subscribe-form" name="mc-embedded-subscribe-form" class="validate" target="_blank" novalidate>
-          <div id="mc_embed_signup_scroll">
-      	<label for="mce-EMAIL">Join the newsletter</label> <br>
-      	<input type="email" value="" name="EMAIL" class="email" id="mce-EMAIL" placeholder="Enter your email address‍" required>
-          <!-- real people should not fill this in and expect good things - do not remove this or risk form bot signups-->
-          <div style="position: absolute; left: -5000px;" aria-hidden="true"><input type="text" name="b_5b441666cd710919cbcaa86e8_d37fc91c90" tabindex="-1" value=""></div>
```

Comment:
So what's happening here is that they're placing a hidden input field in the form, but moving it off the page (`position: absolute; left: -5000px`) so that a human being won't see it. But a bot doesn't look at the page, it just reads the html, so unless the bot is sophisticated it will assume this field should have something in it and fill it in. So if mailchimp gets the result of this form and the form-filler has filled in this field then it knows that this is not a human but a bot and it can ignore the form.